components/ingest-service/integration_test/project_update_actions_test.go

```diff
@@ -337,3 +344,95 @@ func TestProjectUpdateActionsPainlessElasticsearchScript(t *testing.T) {
 			})
 	}
 }
+
+// Test if action are being updated when a project update is ran.
+//
+// Add action with default project tag
+// run update
+// test if the action project tag was update
+func TestProjectUpdateRunsOnActions(t *testing.T) {
+
+	projectID := "1e27fff8-c78b-4f11-9a2a-ae993a642c83"
+	updatedProjectTag := "project9"
+	var eventsSent []*automate_event.EventMsg
+	localSuite := NewLocalSuite(t)
+	defer localSuite.GlobalTeardown()
+
+	localSuite.eventServiceClientMock.EXPECT().Publish(gomock.Any(), gomock.Any()).AnyTimes().DoAndReturn(
+		func(ctx context.Context, in *automate_event.PublishRequest) (*automate_event.PublishResponse, error) {
+			eventsSent = append(eventsSent, in.Msg)
+			return &automate_event.PublishResponse{}, nil
+		})
+	localSuite.projectsClient.EXPECT().ListProjectRules(gomock.Any(), gomock.Any()).AnyTimes().Return(
+		&iam_v2.ProjectCollectionRulesResp{
+			ProjectRules: map[string]*iam_v2.ProjectRules{
+				updatedProjectTag: {
+					Rules: []*iam_v2.ProjectRule{
+						{
+							Type: iam_v2.ProjectRuleTypes_EVENT,
+							Conditions: []*iam_v2.Condition{
+								{
+									Type:   iam_v2.ProjectRuleConditionTypes_CHEF_ORGS,
+									Values: []string{"org1"},
+								},
+							},
+						},
+					},
+				},
+			},
+		}, nil)
+
+	// Add action with default project tag
+	actions := []iBackend.InternalChefAction{
+		{
+			OrganizationName: "org1",
+			Projects:         []string{"old_tag"},
+			RecordedAt:       time.Now(),
+		},
+	}
+	suite.IngestActions(actions)
+	defer suite.DeleteAllDocuments()
+
+	event := &automate_event.EventMsg{
+		EventID:   "5462ebf6-3b5b-40f8-b308-fda4745ce49f",
+		Type:      &automate_event.EventType{Name: automate_event_type.ProjectRulesUpdate},
+		Published: ptypes.TimestampNow(),
+		Data: &_struct.Struct{
+			Fields: map[string]*_struct.Value{
+				project_update_tags.ProjectUpdateIDTag: {
+					Kind: &_struct.Value_StringValue{
+						StringValue: projectID,
+					},
+				},
+			},
+		},
+	}
+
+	// run update
+	_, err := localSuite.EventHandlerServer.HandleEvent(context.Background(), event)
+	assert.NoError(t, err)
+
+	complete := false
+	// Wait for job to complete
+	for !complete {
+		time.Sleep(time.Millisecond * 100)
+
+		for _, event := range eventsSent {
+			assert.Equal(t, event.Type.Name, automate_event_type.ProjectRulesUpdateStatus)
+			if event.Data.Fields["Completed"].GetBoolValue() {
+				complete = true
+			}
+		}
+	}
+
+	suite.RefreshIndices(suite.Indices()...)
+
+	// Test if the action was updated
+	actualActions, err := suite.GeActions(100)
```

Comment:
should this say GetActions? (not `GeActions`)

Reply:
😆 Yes it should. I wonder how long that has been in there like that.